src/start/input_cp2k_motion.F

```diff
@@ -869,16 +869,20 @@ RECURSIVE SUBROUTINE create_geoopt_section(section, location, label, description
       ! collects keywords related to Space Group Symmetry
       CALL keyword_create( &
          keyword, __LOCATION__, name="KEEP_SPACE_GROUP", &
-         description="Keep the space group symmetry operation during optimization.", &
-         usage="KEEP_SPACE_GROUP yes", &
+         description="Detect space group of atoms and preserve during optimization."// &
```

Comment:
Please add a space at the end of each continued line. The `//` operator just concatenates strings.

Reply:
@oschuett Done.